### PR TITLE
Fixes #7 and #34

### PR DESCRIPTION
**Pull request by MatthieuBonne**
I also added some comments to help.

### CODE REVIEW

File: website/api.php [hunk 1]
```diff
@@ -246,7 +246,7 @@ function validateSetupWB($setup,$hero,$fols,$mode) {
         echo json_encode(array("success"=>false,"error"=>"Too much followers"));
         return false;
     }
-    if ($units>6) {
```
Comment (Simoris) [inline website/api.php:249]: can this even be above 6? Isn't it only incremented in the for-loop starting a line 215, so a maximum of 6 times?

File: website/api.php [hunk 2]
```diff
@@ -482,22 +481,24 @@ function doWB($bid,$uid,$damage,$wbhp,$isSuper) {
...
-                $wb = $WB[rand(0,count($WB)-1)];
-                for ($i=0; $i<count($ids); ++$i) {
-                    if ($ids[$i]<7) $wb=$WB[$i];
-                }
+				do { // if we need a NH, prevent bornag from spawning
+					$wb = $WB[rand(0,count($WB)-1)];
+					for ($i=0; $i<count($ids); ++$i) {
+						if ($ids[$i]<7) $wb=$WB[$i];
+					}
+				} while (!$forceNH || $wb==186);
```
Comment (Simoris) [inline website/api.php:496]: This suspiciosly looks like a neverending loop unless there is a forced NH. I think it should be
```suggestion
				} while ($forceNH && $wb==186);
```

File: website/api.php (hunk 1 above)
Comment (MatthieuBonne) [reply website/api.php:249]: Agreed

File: website/api.php (hunk 2 above)
Comment (MatthieuBonne) [reply website/api.php:496]: Doh, I wanted to write || $wb!=186, but ofc your code is more readable, thx.

**Pull request merged**